tests/fallback/fallback_test.go

```diff
@@ -39,6 +39,11 @@ var _ = Describe("cOS booting fallback tests", func() {
 	}
 
 	Context("image is corrupted", func() {
+		BeforeEach(func() {
+			if s.GetArch() == "arm64" {
+				Skip("Tests not supported on arm64 architecture")
```

Comment:
:+1:

Reply:
I was about to switch to just run smoke/upgrades on arm64.. fallback at this point is very focused, but not sure. smoke at least give a good assessment and upgrades tests that upgrades are working at least

either way looks good to me

Reply:
if you want I can drop that in this PR directly

Reply:
I directly dropped the fallback tests for arm64